server/server_test.go

```diff
@@ -591,10 +589,10 @@ func (cli *testServerClient) runTestLoadDataAutoRandomWithSpecialTerm(t *testing
 }
 
 func (cli *testServerClient) runTestLoadDataForListPartition(t *testing.T) {
-	path := "/tmp/load_data_list_partition.csv"
-	defer func() {
-		_ = os.Remove(path)
-	}()
+	f, err := os.CreateTemp("", "load_data_list_partition.csv")
+	require.NoError(t, err)
+	defer os.Remove(f.Name())
```

Comment:
it raises ```Unhandled error```.

Reply:
The original is the way to make lint happy.

Reply:
`make lint` ... there is no error
Or is someone removed it?

Reply:
Goland will raise the warning. I think it may be default config.